tests/sqllogictests/suites/base/05_ddl/05_0000_ddl_create_tables.test

```diff
@@ -140,10 +140,6 @@ CREATE TABLE db2.test5(a Varchar null, y Varchar null) ENGINE=fuse AS SELECT b F
 statement error 1006
 CREATE TABLE db2.test5(a Varchar null, y Varchar null) ENGINE=fuse AS SELECT a, b, a FROM db1.test1
 
-
-statement error 1065
-create table db2.test6(id Int8, created timestamp  DEFAULT CURRENT_TIMESTAMP)
```

Comment:
This can't be deleted, we need to support this feature cause it's widely used.

Reply:
We didn't support literal CURRENT_TIMESTAMP yet. Is it a good idea to support both the literal and the function version?